README.md

```diff
@@ -0,0 +1,77 @@
+This is a dashboard for tracking outstanding bugs in **Ironic**.
```

Comment:
Given we have a PR up to make this generic, perhaps we should indicate it's for tracking bugs in launchpad projects generally? Curious what @dtantsur thinks about that

Reply:
SGTM

Reply:
Another comment: this file is missing a heading.